common/shared/src/main/java/app/coronawarn/server/common/shared/util/SecurityUtils.java

```diff
@@ -49,7 +49,11 @@ public static void ecdsaSignatureVerification(final byte[] encodedSignature, fin
     signatureVerification.update(content);
 
     if (!signatureVerification.verify(encodedSignature)) {
-      throw new SignatureException("Signature verification failed");
+      throw new SignatureException("Signature verification " + signatureVerification
```

Comment:
Maybe we want to use the algorithm instead of the object itself?

Reply:
Good idea. Done :-)